themes/ddbasic/sass/bibliofil/_overides.scss

```diff
@@ -0,0 +1,438 @@
+//
+// Styling of site header
+
+//@import '../base.scss';
+
+//Overid the background for body
+
+html,
+body,
+.ding2-site-template {
+  background-color: $bibliofil;
+  color: $white;
+}
+
+h1,
+.node-ding-news.node-full h1,
+.node-ding-event.node-full h1 {
+  color: $white;
+}
+
+//Overid the background for content and messages
+.content-wrapper {
+  .pane-content {
+    .default-panel-layout {
+      .left-and-main-content {
+        background-color: $bibliofil-dark;
+        background-image: none;
+      }
+    }
+  }
+
+  .panel-pane.system-messages,
+  .panel-pane .messages.error {
+    background-color: $bibliofil-dark;
+    background-image: none;
+  }
+}
+ div.panels-ipe-placeholder {
+    background-color: $bibliofil-dark;
+    background-image: none;
+  }
+
+.content-wrapper {
+  .pane-content {
+    .frontpage-layout {
+      .panel-pane{
+    background-color: $bibliofil;
+    background-image: none;
+
+      }
+    }
+  }
+}
+
+//Overid the background and color for input
+.panel-pane.pane-import {
+  input[type=text] {
+    background-color: $white;
+    color: $bibliofil-button;
+  }
+}
+
+//Overid event and news background
+
+.node-ding-news.node-full,
+.node-ding-event.node-full,
+.pane-views-panes.pane-ding-library-ding-library-libraries-list {
+  background-color: $bibliofil;
+  background-image: none;
+}
+
+//Overid the background and color for input
+
+.pane-views-exp-ding-event-ding-event-list .form-item input {
+  background-color: $white;
+  color:  $text-input;
+}
+
+//Overid the background for submint button
```

Comment:
Start with a space, end with a dot. Override is spelled with "e" in the end. Submit is misspelled.